ingest/src/main/java/com/microsoft/azure/kusto/ingest/QueuedIngestClientImpl.java

```diff
@@ -226,7 +226,7 @@ public IngestionResult ingestFromStream(StreamSourceInfo streamSourceInfo, Inges
                     shouldCompress);
             String blobPath = container.getContainer().getBlobContainerUrl() + "/" + blobName + container.getSas();
             BlobSourceInfo blobSourceInfo = new BlobSourceInfo(
-                    blobPath, 0); // TODO: check if we can get the rawDataSize locally - maybe add a countingStream
+                    blobPath, 0, streamSourceInfo.getSourceId()); // TODO: check if we can get the rawDataSize locally - maybe add a countingStream
```

Comment:
What does this do? did we send it wrong so far?

Reply:
yes